invenio_vocabularies/contrib/affiliations/facets.py

```diff
@@ -14,14 +14,22 @@
 class AffiliationsLabels(VocabularyLabels):
     """Fetching of vocabulary labels for facets."""
 
-    def __init__(self, vocabulary, cache=False, service_name=None):
-        """Initialize the labels."""
+    def __init__(self, vocabulary, cache=True, cache_for=3600, service_name=None):
```

Comment:
ultra-minor: I thought it was cache "for this vocabulary" like a key, maybe `ttl` or `expiration`.

Reply:
good point, I will change it